src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    private Ball ball = new Ball();
```

Comment:
this should be a local variable, your method getRandomBall() is not creating a new ball - it changes properties of this ball, and return it
if you call this method a few times, stores those balls, then output it into console - you will see that they are all the same